frontend/public/components/utils/resource-link.jsx

```diff
@@ -59,7 +59,7 @@ export const resourceObjPath = (obj, kind) => resourcePath(kind, _.get(obj, 'met
 
 export const ResourceLink = connectToModel(
   ({className, displayName, inline = false, kind, kindsInFlight, linkTo = true, name, namespace, hideIcon, title}) => {
-    if (!kind && kindsInFlight) {
+    if (!kind) {
```

Comment:
We shouldn't check `kindsInFlight` here since an undefined kind will result in a runtime error later.